lib/payday/invoice.rb

```diff
@@ -13,9 +13,9 @@ def initialize(options =  {})
       self.ship_to = options[:ship_to] || nil
       self.notes = options[:notes] || nil
       self.line_items = options[:line_items] || []
-      self.shipping_rate = options[:shipping_rate] || nil
+      self.shipping_rate = options[:shipping_rate] || 0
```

Comment:
shouldn't these be doubles `0.0`?

Reply:
it says that it can't omit precision for a Float, bit also `BigDecimal.new(0)` returns 0.0 of type BigDecimal

Reply:
I don't understand :(

Reply:
I though you asked about why not passing 0.0 instead of 0 as BigDecimal argument.

This gem is used to print some financial information but also internally it sums up the individual line items and displays a totals, so I think we need that precision.